apps/web/pages/settings/teams/[id]/index.tsx

```diff
@@ -95,7 +95,7 @@ export function TeamSettingsPage() {
               )
             }>
             {!!errorMessage && <Alert className="-mt-24 border" severity="error" title={errorMessage} />}
-            {team && (
+            {/* {team && (
               <>
                 <div className="block sm:flex md:max-w-5xl">
                   <div className="w-full ltr:mr-2 rtl:ml-2 sm:w-9/12">
```

Comment:
GH is being weird and not letting me comment the whole block. Because this is a V1 screen I just commented this out since it was giving type errors